src/components/DaySlot.jsx

```diff
@@ -0,0 +1,225 @@
+import React, { PropTypes, Component } from 'react'
+import { findDOMNode } from 'react-dom'
+import cn from 'classnames';
+
+import SelectableTimeGutter from '../containers/SelectableTimeGutter.jsx'
+import EventRow from '../EventRow.jsx'
+
+import dates from '../utils/dates';
+import { isSelected } from '../utils/selection';
+import { notify } from '../utils/helpers';
+import { accessor, elementType } from '../utils/propTypes';
+import { accessor as get } from '../utils/accessors';
+import localizer from '../localizer'
+
+export default class DaySlot extends Component {
+  static propTypes = {
+    formatter: PropTypes.func,
```

Comment:
it is the text that is displayed inside the selection box.  A user may choose to render this (which I will do) as "15 minute major jury" instead of "10:15-10:30" using this callback